src/BinSkim.Driver/AnalyzeCommand.cs

```diff
@@ -134,6 +137,24 @@ public override int Run(AnalyzeOptions analyzeOptions)
                 : result;
         }
 
+        internal static SarifLog ReadSarifLog(IFileSystem fileSystem, AnalyzeOptions analyzeOptions)
```

Comment:
>internal static SarifLog ReadSarifLog [](http://example.com/codeflow?start=8&length=37)

Creating this helper to facilitate multiple versions of the SARIF file.

This will also enable us to test the SARIF Load process

Reply:
just a thought,
A Sarif file read based on the version can be a common method in Sarif SDK as well, with a parameter,
since the code will be the same if user is BinSkim or other tools, so they don't need to figure out and write the same

Reply:
learning while reading PR :)
I see there is SarifVersion has a SarifVersion.Unknown 
do we need to add it in the if else

Reply:
I think that would be a good idea, but we would have to think if this is a public api that we want to expose.

For now, the only exposed api we have that helps read the SARIF is the ReadSarifFile, which is available for command classes

I haven't seen any applicability for that yet (SarifVersion.Unknown).